### ISSUE TITLE
React sample page fails with many exports not found

### ISSUE THREAD

**Issue by mdeselincourt** [aws-amplify/amplify-ui#1093]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-adminui/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-adminui/blob/main/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### App Id

d2yvia01jvycqr

### Region

eu-west-1

### Environment name

dev

### Amplify CLI Version

7.6.5

### If applicable, what version of Node.js are you using?

v16.13.1

### What operating system are you using?

Windows

### Browser type?

Chrome and others

### Describe the bug

I have followed the tutorial - created a default react app, which works, then used amplify to register it as an app, which works.

I am following the instructions at https://ui.docs.amplify.aws/

I have run **npm i @aws-amplify/ui-react aws-amplify** multiple times, and also with the -S option just in case.

I have created the App.js and theme.js files instructed by https://ui.docs.amplify.aws/

```ERROR in ./src/App.js 7:0-42
Module not found: Error: Can't resolve '@aws-amplify/ui-react/styles.css' in 'C:\Users\Mike Local\github\spacebiz3-reamp\spacebiz3-reamp\src'
 @ ./src/index.js 7:0-24 17:33-36

ERROR in ./src/App.js 16:6-14
export 'useTheme' (imported as 'useTheme') was not found in '@aws-amplify/ui-react' (possible exports: AmplifyAmazonButton, AmplifyAuth0Button, AmplifyAuthContainer, AmplifyAuthFields, AmplifyAuthenticator, AmplifyButton, AmplifyChatbot, AmplifyCheckbox, AmplifyCodeField, AmplifyConfirmSignIn, AmplifyConfirmSignUp, AmplifyContainer, AmplifyCountryDialCode, AmplifyEmailField, AmplifyFacebookButton, AmplifyFederatedButtons, AmplifyFederatedSignIn, AmplifyForgotPassword, AmplifyFormField, AmplifyFormSection, AmplifyGoogleButton, AmplifyGreetings, AmplifyHint, AmplifyIcon, AmplifyIconButton, AmplifyInput, AmplifyLabel, AmplifyLink, AmplifyLoadingSpinner, AmplifyNav, AmplifyOauthButton, AmplifyPasswordField, AmplifyPhoneField, AmplifyPhotoPicker, AmplifyPicker, AmplifyRadioButton, AmplifyRequireNewPassword, AmplifyS3Album, AmplifyS3Image, AmplifyS3ImagePicker, AmplifyS3Text, AmplifyS3TextPicker, AmplifySection, AmplifySelect, AmplifySelectMfaType, AmplifySignIn, AmplifySignInButton, AmplifySignOut, AmplifySignUp, AmplifyStrike, AmplifyToast, AmplifyTooltip, AmplifyTotpSetup, AmplifyUsernameField, AmplifyVerifyContact, withAuthenticator)
 @ ./src/index.js 7:0-24 17:33-36

export 'Card' (imported as 'Card') was not found in '@aws-amplify/ui-react' (possible exports: AmplifyAmazonButton, AmplifyAuth0Button, AmplifyAuthContainer, AmplifyAuthFields, AmplifyAuthenticator, AmplifyButton, AmplifyChatbot, AmplifyCheckbox, AmplifyCodeField, AmplifyConfirmSignIn, AmplifyConfirmSignUp, AmplifyContainer, AmplifyCountryDialCode, AmplifyEmailField, AmplifyFacebookButton, AmplifyFederatedButtons, AmplifyFederatedSignIn, AmplifyForgotPassword, AmplifyFormField, AmplifyFormSection, AmplifyGoogleButton, AmplifyGreetings, AmplifyHint, AmplifyIcon, AmplifyIconButton, AmplifyInput, AmplifyLabel, AmplifyLink, AmplifyLoadingSpinner, AmplifyNav, AmplifyOauthButton, AmplifyPasswordField, AmplifyPhoneField, AmplifyPhotoPicker, AmplifyPicker, AmplifyRadioButton, AmplifyRequireNewPassword, AmplifyS3Album, AmplifyS3Image, AmplifyS3ImagePicker, AmplifyS3Text, AmplifyS3TextPicker, AmplifySection, AmplifySelect, AmplifySelectMfaType, AmplifySignIn, AmplifySignInButton, AmplifySignOut, AmplifySignUp, AmplifyStrike, AmplifyToast, AmplifyTooltip, AmplifyTotpSetup, AmplifyUsernameField, AmplifyVerifyContact, withAuthenticator)

and so on
```

### Expected behavior

I expected the sample React page to render

### Reproduction steps

Create default react app
Install aws-amplify @aws-amplify/ui-react@1.x.x as per Amplify tutorial
Follow instructions on https://ui.docs.amplify.aws/  to create a new App.js and theme.js
npm start

**Comment by reesscot** [comment]
Hi @mdeselincourt,

It sounds like you may have imported the 1.x version of the library. Can you please try updating to the 2.x version of the library?
```
npm install @aws-amplify/ui-react@latest
```

**Comment by jmcopas** [comment]
I get a similar issue to this when using 'withAuthenticator' component (https://docs.amplify.aws/lib/auth/getting-started/q/platform/js/#option-1-use-pre-built-ui-components):

`ERROR in ./node_modules/@aws-amplify/ui-react/dist/esm/index.js 34536:9-17
export 'useId' (imported as 'n9') was not found in 'react' (possible exports: Children, Component, Fragment, Profiler, PateElement, createFactory, createRef, forwardRef, isValidElement, lazy, memo, useCallback, useContext, useDebugValue, useEffect, useImperativeHandle, useLayoutEffect, useMemo, useReducer, useRef, useState, version)
 @ ./src/components/login.tsx 5:0-58 36:21-38
 @ ./src/App.tsx 7:0-39 59:35-40
 @ ./src/index.tsx 7:0-24 11:33-36`

`npm ls @aws-amplify/ui-react`
`@aws-amplify/ui-react@2.1.7`

This is in a vanilla create react app **(with TypeScript)** build.

**Comment by wlee221** [comment]
@jmcopas, this has been resolved in #1100 and duplicate of #1154.

**Comment by wlee221** [comment]
I'll go ahead and close this issue as problems mentioned here have been addressed, but feel free to open a new one if similar problem persists.
